### PR TITLE
#Upgrade our issue template

### PR DESCRIPTION
**Pull request by lucaoskaique**

### CODE REVIEW

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Hey, we have few more changes required. For example you can ask for the library version, php version and code of conduct on all the templates. We have a first approved PR please take that as reference
https://github.com/utopia-php/framework/pull/28

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      placeholder: "It should ..."
+  - type: textarea
+    id: actual-behavior
+    validations:
+      required: true
+    attributes:
+      label: "👎 Actual Behavior"
+      description: "What did actually happen? Add screenshots, if applicable."
+      placeholder: "It actually ..."
+  - type: dropdown
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:34]: Let's also add Utopia CLI version option, see the versions from releases and keep others option as well

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Great work @lucaoskaique Please consider looking at the approved PR add missing things and make template consistent for code of conduct.

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 1 above)
Comment (lucaoskaique) [reply .github/ISSUE_TEMPLATE/bug.yaml:34]: Got it

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    id: utopia-version
+    attributes:
+      label: "🎲 Utopia Framework version"
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:37]:
```suggestion
      label: "🎲 Utopia CLI version"

```

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Great work. Just a small fix.

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  - type: checkboxes
+    id: no-duplicate-issues
+    attributes:
+      label: "👀 Have you spent some time to check if this issue has been raised before?"
+      description: "Have you Googled for a similar issue or checked our older issues for a similar bug?"
+      options:
+        - label: "I checked and didn't find similar issue"
+          required: true
+  - type: checkboxes
+    id: read-code-of-conduct
```
Comment (lohanidamodar) [inline .github/ISSUE_TEMPLATE/bug.yaml:89]: The format of code of conduct field is different here. please address that

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
Everything is great @lucaoskaique just small fix for consistency.

**Comment by christyjacob4** [comment]
@lucaoskaique closing this PR for now since there has been no activity in quite a long time. Feel free to reopen it when you're ready to start working on it once again. 🙂

**Pull request closed without merging**